jerry-core/ecma/base/ecma-gc.c

```diff
@@ -88,88 +88,45 @@ static void ecma_gc_mark (ecma_object_t *object_p);
 static void ecma_gc_sweep (ecma_object_t *object_p);
 
 /**
- * Get GC reference counter of the object.
- */
-static uint32_t
-ecma_gc_get_object_refs (ecma_object_t *object_p) /**< object */
-{
-  JERRY_ASSERT (object_p != NULL);
-
-  return (uint32_t) JRT_EXTRACT_BIT_FIELD (uint64_t, object_p->container,
-                                           ECMA_OBJECT_GC_REFS_POS,
-                                           ECMA_OBJECT_GC_REFS_WIDTH);
-} /* ecma_gc_get_object_refs */
-
-/**
- * Set GC reference counter of the object.
- */
-static void
-ecma_gc_set_object_refs (ecma_object_t *object_p, /**< object */
-                         uint32_t refs) /**< new reference counter */
-{
-  JERRY_ASSERT (object_p != NULL);
-
-  object_p->container = JRT_SET_BIT_FIELD_VALUE (uint64_t, object_p->container,
-                                                 refs,
-                                                 ECMA_OBJECT_GC_REFS_POS,
-                                                 ECMA_OBJECT_GC_REFS_WIDTH);
-} /* ecma_gc_set_object_refs */
-
-/**
  * Get next object in list of objects with same generation.
  */
-static ecma_object_t *
+static inline ecma_object_t *
```

Comment:
That is a different reason. Getting and setting is used for incrementing and decrementing reference counters, but that is an inefficient way to do that (lots of shifting, masking). After the refcounter updates are optimized, those functions are not used anymore, so they are removed.